### DISCUSSION TITLE
Tones / downmix

### DISCUSSION THREAD

**Discussion by 0x0fe** [pschatzmann/arduino-audio-tools#995] [Q&A] [answered]
In ADF there is one (single) example of downmix, which allows to play tones (or other audio assets) together, the downmix implementation also provide some automatic "voiceover" effect by smoothly decreasing the level of background audio and increasing the level of the "tone" while it plays, and reverting at the end.
While the voiceover feature is not a necessity, i find that being able to play tones over the background audio is quite an important feature, in particular for UI notifications. 
Is there a way to achieve this in arduino-audio-tools?

**Comment by pschatzmann** [comment, marked as answer]
Did you read the chapter for Mixing ? 
All you need to do is to adjust the weights over time  (=after each copy step).

If you think that this should have it's own class, this should be pretty easy to implement. 
Feel free to submit a pull request.
